policy-controller/grpc/src/outbound.rs

```diff
@@ -529,3 +538,20 @@ fn default_queue_config() -> outbound::Queue {
         ),
     }
 }
+
+fn convert_duration(
```

Comment:
Having a curried function here feels overly fancy.  Is this just so that we can pass it directly to `and_then`?  This would read more easily to me to just use `try_into`, `map_err`, and `ok` at the call-sites, even if it is more repetitive.